src/mem/ruby/protocol/MOESI_AMD_Base-dir.sm

```diff
@@ -1397,22 +1400,33 @@ machine(MachineType:Directory, "AMD Baseline protocol")
     p_popRequestQueue;
   }
 
-  transition(U, VicDirty, BL) {L3TagArrayRead} {
+  transition(U, VicDirty, U) {L3TagArrayRead, L3DataArrayWrite} {
```

Comment:
The 3rd transition `, U` is not necessary here (it will remain in the same state)